cmd/image-builder/main_test.go

```diff
@@ -123,142 +124,187 @@ func Test_parseVariable(t *testing.T) {
 	}
 }
 
-func Test_validateOptions(t *testing.T) {
-	tc := []struct {
-		name      string
-		expectErr bool
-		opts      options
-	}{
-		{
-			name:      "parsed config",
-			expectErr: false,
-			opts: options{
-				context:    "directory/",
-				name:       "test-image",
-				dockerfile: "dockerfile",
-				configPath: "config.yaml",
-			},
+var _ = Describe("Image Builder", func() {
+	DescribeTable("Test validate options",
+		func(options options, expectedError bool) {
+			err := validateOptions(options)
+			if err != nil && !expectedError {
+				Fail(fmt.Sprintf("caught error, but didn't want to: %v", err))
+			}
+			if err == nil && expectedError {
+				Fail("didn't catch error, but wanted to")
+			}
```

Comment:
```suggestion
			if !expectedError {
				Expect(err).NotTo(HaveOccured(),fmt.Sprintf("caught error, but didn't want to: %v", err))
			}
			if expectedError {
				Expect(err).To(HaveOccured(), "didn't catch error, but wanted to")
			}
```
Let's use gomega matchers to evaluate `err` expected values.